conf/layer.conf

```diff
@@ -30,5 +30,5 @@ LICENSE_PATH += "${LAYERDIR}/licenses"
 # Enable early start of Thunder
 DISTRO_FEATURES_append = " compositor thunder"
 
-#Added for testing purpose
-PACKAGECONFIG_pn-wpeframework-buildtype = " debugoptimized"
+# FIXME: has to be passed/selected based on the system config requirement
```

Comment:
This is fine for now, but could you please move that to the conf/distro/include/thunder_debug.conf? Additionally please a comment so that if you want to build wpa_supplicant in the debug mode -R thunder_debug.conf should be used?